tenders/hvt/solo5-hvt-configure

```diff
@@ -88,7 +88,7 @@ add_module ()
 }
 
 SOLO5_HVT_CFLAGS=
-SOLO5_HVT_LDFLAGS=
+SOLO5_HVT_LDFLAGS=-static
```

Comment:
Why is this required? `solo5-hvt` is not usually built statically?

Reply:
Reverted, this is and artifact from the LDFLAGS shuffle.